xrpl4j-keypairs/src/main/java/org/xrpl/xrpl4j/keypairs/Secp256k1KeyPairService.java

```diff
@@ -106,23 +108,24 @@ private BigInteger deriveScalar(UnsignedByteArray seed, Optional<Integer> discri
   @Override
   public String sign(UnsignedByteArray message, String privateKey) {
     UnsignedByteArray messageHash = HashUtils.sha512Half(message);
-    ECDSASignature signature = createEcdsaSignature(messageHash, new BigInteger(privateKey, 16));
+    EcdsaSignature signature = createEcdsaSignature(messageHash, new BigInteger(privateKey, 16));
```

Comment:
Very minor nit, but I think this should be `EcDsaSignature`

Reply:
Sure thing